src/scenes/Scene.js

```diff
@@ -33,3 +33,18 @@ THREE.Scene.prototype.copy = function ( source, recursive ) {
 	return this;
 
 };
+
+THREE.Scene.prototype.toJSON = function ( meta ) {
+
+	var data = THREE.Object3D.prototype.toJSON.call( this, meta );
+	
+	if ( this.fog != null ) {
```

Comment:
Note: using loose equality to check for null instead of strict equality checking for undefined as is the practice in the codebase. This is because fog is actually null when parsing the editor scene and this way we catch both null and undefine.